### PR TITLE
chore(dependencies): Autobump Front50 Version

### PR DESCRIPTION
**Pull request by link108**

### CODE REVIEW

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/canary/google/GoogleCanaryAccountValidator.java [hunk 1]
```diff
@@ -40,6 +44,7 @@
   private String halyardVersion;
 
   private Registry registry;
+  private Storage googleCloudStorage;
```
Comment (german-muzquiz) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/canary/google/GoogleCanaryAccountValidator.java:47]: How is this field initialized? I assume it should be different for each `GoogleCanaryAccount`, which is passed in `validate`

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/persistentStorage/GCSValidator.java [hunk 2]
```diff
@@ -36,6 +40,7 @@
   @Autowired private Registry registry;
 
   @Autowired TaskScheduler taskScheduler;
+  private Storage googleCloudStorage;
```
Comment (german-muzquiz) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/persistentStorage/GCSValidator.java:43]: Same comment as above

**Comment by link108** [comment]
@german-muzquiz thank you for the feedback, I've pushed out an update if/when you have a chance to look at it 👍

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/canary/google/GoogleCanaryAccountValidator.java [hunk 3]
```diff
@@ -41,6 +49,8 @@
 
   private Registry registry;
 
+  @Autowired Storage googleCloudStorage;
```
Comment (german-muzquiz) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/canary/google/GoogleCanaryAccountValidator.java:52]:
```suggestion

```

**Review by german-muzquiz — APPROVED** [review]
Approved with a small suggestion to delete unused autowired variable

**Pull request merged**